tests/unit/test_sqlite_edge_cases.py

```
@@ -194,7 +200,9 @@ async def test_backup_with_no_write_permissions(self, tmp_path: Path) -> None:
         assert not db_path.exists()
 
     @pytest.mark.asyncio
-    async def test_backup_with_disk_full_error(self, tmp_path: Path) -> None:
+    async def test_backup_with_disk_full_error(
+        self, tmp_path: Path, sqlite_backend_factory
```

Ruff (0.13.2)

Unused method argument: `sqlite_backend_factory`

(ARG002)